cmd/lncli/commands.go

```diff
@@ -289,10 +289,11 @@ var sendCoinsCommand = cli.Command{
 		},
 		cli.BoolFlag{
 			Name: "sweepall",
-			Usage: "if set, then the amount field will be ignored, " +
-				"and the wallet will attempt to sweep all " +
-				"outputs within the wallet to the target " +
-				"address",
+			Usage: "if set, then the amount field should be " +
+				"unset. This indicates that the wallet will" +
```

Comment:
nit: move space to end of this line to be consistent and easier to review (or break a word earlier if line becomes too long otherwise).